README.md

```diff
@@ -231,9 +231,9 @@ If an Observer fails to meet these expectations they can be excluded from future
 * [Collaboration Space Progression](./collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md)
 * [Working Groups](./governance/WORKING_GROUPS.md)
 
-#### Other 
+#### Community Fund 
```

Comment:
Is this https://github.com/openjs-foundation/community-fund repository still relevant?